src/iso19111/factory.cpp

```diff
@@ -3114,9 +3114,9 @@ std::string
 DatabaseContext::suggestsCodeFor(const common::IdentifiedObjectNNPtr &object,
                                  const std::string &authName,
                                  bool numericCode) {
-    const char *tableName = "";
+    const char *tableName = "prime_meridian";
```

Comment:
Modifying a const char * ?
(I see it is not changing the chars, but the pointee)
What was the checker complain to change it now?
Would it make more sense to use an std::string?

Reply:
the checker complained about the initial value never being read and always modified. This is tiny more bit performant (and contribute to binary size reduction) to use a const char* here given that it is a fixed set of strings, although that shouldn't matter in practice.